libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -922,9 +922,11 @@ AP_InertialSensor::detect_backends(void)
 #endif
 
     uint8_t probe_count = 0;
+    (void)probe_count;
```

Comment:
Alternatively we could have `HAL_INS_ENABLED`.  If you want the INS lib in AP_Periph you have to override that - and if you subsequently don't allow backends you'd die.  But at least you'd have an idea of what's going on...